scripts/workers/submission_worker.py

```diff
@@ -750,6 +751,25 @@ def load_challenge_and_return_max_submissions(q_params):
     return maximum_concurrent_submissions, challenge
 
 
+def increment_and_push_metrics_to_statsd(body, queue_name):
+    try:
+        submission_pk = json.loads(body)["submission_pk"]
```

Comment:
message body is loaded from yaml. Refer [this](https://github.com/Cloud-CV/EvalAI/pull/3542/files#diff-ae6210f157b1ef66d873b08fe30f65f36c92c1e3d8aa61e3c8580910f5f06695L690-L691)